.changeset/erpnext-initial.md

```diff
@@ -0,0 +1,14 @@
+---
+"@openfn/language-erpnext": minor
+---
+
+Create ERPNext adaptor with CRUD and list operations
+
+- Implement create, read, update, and deleteRecord operations
+- Add getList operation with filters, pagination, and field selection
+- Add getCount operation for counting documents
+- Support API Key/Secret token-based authentication
+- Include comprehensive JSDoc documentation with examples
+- Add full test coverage with 17 passing unit tests
+- Include official ERPNext logos (256x256 and 512x190)
+- Integration tested against live ERPNext instance
```

Comment:
Hey @rubailly i can't seem to find integration test in pr, If you have them can you share the code in this pr. If you can just remove this line